docs/proxy.md

```diff
@@ -0,0 +1,31 @@
+# Configuring egress proxy for AWS Load Balancer Operator
+
+If a cluster wide egress proxy is configured on the OpenShift cluster, OLM automatically update all the operators' deployments with `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` environment variables.
+
+## Trusted Certificate Authority
+
+### Running operator
+Follow the instructions below to let AWS Load Balancer Operator trust a custom Certificate Authority (CA). The operator's OLM subscription has to be already created.
+The operator's deployment doesn't have to be ready though.
+
+1. Create the configmap containing the CA bundle in `aws-load-balancer-operator` namespace.
+    ```bash
+    oc -n aws-load-balancer-operator create configmap trusted-ca
+    oc -n aws-load-balancer-operator label cm trusted-ca config.openshift.io/inject-trusted-cabundle=true
+    ```
+
+2. Consume the created configmap in AWS Load Balancer Operator's deployment by updating its subscription:
+
+    ```bash
+    oc -n aws-load-balancer-operator patch subscription aws-load-balancer-operator --type='merge' -p '{"spec":{"config":{"volumes":[{"name":"trusted-ca","configMap":{"name":"trusted-ca"}}],"volumeMounts":[{"name":"trusted-ca","mountPath":"/etc/pki/tls/certs/albo-tls-ca-bundle.crt","subPath":"ca-bundle.crt"}]}}}'
+    ```
+
+3. Wait for the operator deployment to finish the rollout and verify that CA bundle is added:
+
+    ```bash
+    oc -n aws-load-balancer-operator exec deploy/aws-load-balancer-operator-controller-manager -c manager -- ls -l /etc/pki/tls/certs/albo-tls-ca-bundle.crt
+
+    -rw-r--r--. 1 root 1000690000 5875 Jan 11 12:25 /etc/pki/tls/certs/albo-tls-ca-bundle.crt
+    ```
+
+    **Note**: The configmap is consumed using [subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) which means that no updates will be seen by the operator's POD. Make sure the operator is restarted every time the configmap contents are changed (`oc rollout restart` command).
```

Comment:
```suggestion
4. Restart the operator:
```bash
oc rollout restart
```
    **Note**: The configmap is consumed using [subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) which means that no updates will be seen by the operator's POD. Make sure the operator is restarted every time the configmap contents are changed.
```

Reply:
The rollout happens automatically after the operator's subscription is patched in the command #2.

Reply:
Yes, maybe we can make "restart" step as optional, since it is just required after user updating the proxy/cluster `spec.trustedCA` and custom CA already configured for ALBO.

Reply:
and the complete command looks like
```
oc -n aws-load-balancer-operator rollout restart deployment/aws-load-balancer-operator-controller-manager
```

Reply:
Ok, now I understood the suggestion. Yes, I think it makes sense to add the restart as the fourth step.